share/spice/yacht_specs/content.handlebars

```diff
@@ -0,0 +1,32 @@
+<!-- Div for the yacht specifications -->
+<div class='thumb'>
+    
+    <div class='image'>
```

Comment:
The image will actually be handled by the template! Please remove this and add a `normalize` block to your `Spice.add()` call:

```js
Spice.add({
  ...
  meta: {
    ..
  },
  normalize: function(data) {
    image: data.result_photo,
    title: data.name,
    subtitle: "Built in " + data.year
  },
  ...
}
```